windows/Dockerfile

```diff
@@ -157,9 +154,10 @@ RUN powershell -C .\install_msys.ps1 -Version $ENV:MSYS_VERSION
 
 RUN ridk install 3
 
-# (32-bit only) Install 32-bit C/C++ compilation toolchain
-RUN if ($Env:TARGET_ARCH -eq 'x86') { ridk enable; bash -c \"pacman -S --needed --noconfirm mingw-w64-i686-binutils mingw-w64-i686-crt-git mingw-w64-i686-gcc mingw-w64-i686-gcc-libs mingw-w64-i686-headers-git mingw-w64-i686-libmangle-git mingw-w64-i686-libwinpthread-git mingw-w64-i686-make mingw-w64-i686-pkg-config mingw-w64-i686-tools-git mingw-w64-i686-winpthreads-git\" }
-RUN if ($Env:TARGET_ARCH -eq 'x86') { [Environment]::SetEnvironmentVariable(\"Path\", [Environment]::GetEnvironmentVariable(\"Path\", [EnvironmentVariableTarget]::Machine) + \";C:\tools\msys64\mingw32\bin;C:\tools\msys64\usr\bin\", [System.EnvironmentVariableTarget]::Machine) }
+# Downgrade gcc due to an incompatibility between gcc 10.3 and go 1.15 and 1.16
+RUN (New-Object System.Net.WebClient).DownloadFile(\"https://s3.amazonaws.com/dd-agent-omnibus/mingw-w64-x86_64-gcc-10.2.0-11-any.pkg.tar.zst\", \"C:/mingw-w64-x86_64-gcc-10.2.0-11-any.pkg.tar.zst\")
+RUN (New-Object System.Net.WebClient).DownloadFile(\"https://s3.amazonaws.com/dd-agent-omnibus/mingw-w64-x86_64-gcc-libs-10.2.0-11-any.pkg.tar.zst\", \"C:/mingw-w64-x86_64-gcc-libs-10.2.0-11-any.pkg.tar.zst\")
```

Comment:
In theory yes, in practice I don't know if it makes sense to spend time fixing builders we don't use (and that we may want to retire in the future)?

Reply:
Yes, either that or re-add the two x86-specific lines for now (to not make the x86 test jobs fail because there's no x86 toolchain available).

Reply:
Oh, I thought we were not building them anymore (since we are not using them anymore in the Agent pipeline anymore). I'll remove the x86 build and test jobs from here.